modules/docs/mdx/8.0-UPGRADE-GUIDE.mdx

```diff
@@ -208,77 +356,28 @@ const StyledBox = styled(Box)({
 <Box fontFamily="monospace" fontSize={16} fontWeight="medium" />
 ```
 
-## General Changes
-
-### Remove Default Imports
-
-As part of some cleanup, we've removed any default imports that where lingering. This change matches
-more closely what we've been doing with new components and systems. As part of this change, the
-components listed below have moved from default imports to named imports.
-
-##### Fonts
-
-- fonts
-
-##### Main Package Components
-
-- Toast
-- Canvas
-- Tooltip
-- TextArea
-- Avatar
-- CountBadge
-- Checkbox
-- FormField
-- LoadingDots
-- Radio
-- SegmentedControl
-- Select
-- SidePanel
-- StatusIndicator
-- Switch
-- Table
-- TextInput
-
-##### Labs Components
-
-- Combobox
-- Drawer
-
-##### Preview Components
-
-- Breadcrumbs
-- ColorPicker
-- Menu
-- Select
-- SidePanel
-
-🤖 The codemod will update imports to be named imports
-
-### Menu (main)
-
-Default depth value of the main menu component has been change to depth 3 (It was depth 1 before).
-Depth update is a visual breaking change but doesn't require any actions from user.
+---
 
 ### Breadcrumbs
 
-Breadcrumbs component has been promoted to `canvas-kit-react` package because now it uses the same
-list system being developed for Tabs, ActionBar which provides the new overflow behavior based on
-utilizing `useBreadcrumbsModel`. Overflow is dynamic and calculated on the container width.
+`Breadcrumbs` has been promoted to the Main package. It now uses the list system from our
+[Collection API](/getting-started/for-developers/resources/collection-api/) which provides new
+overflow behavior based on container width.
```

Comment:
@alanbsmith @RayRedGoose I could use extra eyes on the Breadcrumbs section here in the Upgrade Guide. I fixed a couple errors and did a fair amount of wordsmithing here -- let me know if I made any mistakes or altered the intent of what you wanted to communicate.